src/components/listOfQuestions/index.test.js

```diff
@@ -0,0 +1,124 @@
+import { render, screen, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { Question } from './index.js';
+import { validateQuestionDisplay } from './validations.js';
+import cuid from 'cuid';
+import { set } from 'date-fns';
+
+const questionFactory = ({
+	id = cuid(),
+	timestamp = set(new Date(), {
+		hours: 0,
+		minutes: 0,
+		seconds: 0,
+		milliseconds: 0,
+	}),
+	question = 'Placeholder question',
+	askee = 'Anonymous',
+	status = 'Unanswered',
+} = {}) => ({
+	id,
+	timestamp,
+	question,
+	askee,
+	status,
+});
+
+const noop = () => {};
+const renderQuestion = (
+	question,
+	{ onAccept = noop, onReject = noop } = {}
```

Comment:
If you move the defaults to the component, you wouldn't need these defaults here.